test/pylib/scylla_cluster.py

```diff
@@ -694,6 +694,8 @@ async def add_server(self, replace_cfg: Optional[ReplaceConfig] = None,
             raise RuntimeError("Can't add the server: all servers in the cluster are stopped")
 
         extra_config: dict[str, Any] = config.copy() if config else {}
+        if "maintenance_socket" not in extra_config:
+            extra_config["maintenance_socket"] = "ignore"
```

Comment:
>  test.py: disable maintenance port by default 

Please write motivation in the commit message

Reply:
Removed this commit, since it is not needed to set this value as ignore by default in tests.